src/Helper/EnumValue.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+namespace Torr\Rad\Helper;
+
+abstract class EnumValue
+{
+	/**
+	 * Helper to resolve an enum / string value to a string.
+	 */
+	public static function strict (\BackedEnum|string $value) : string
```

Comment:
The values are supposed to be really short, as we otherwise won't gain anything.

```php
function (\BackedEnum|string|null $value)
{
    $value = EnumValue::get($value);
}
```

needs to be much shorter than

```php
function (\BackedEnum|string|null $value)
{
   $value = $value instanceof \BackedEnum ? $value->value : $value;
}
```

otherwise this method has no purpose

Reply:
I see. Well, that limits our options quite significantly. In that case, I'd say that we just keep it 👍